Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1841,6 +1843,13 @@
     Telecrystal: 12 # WD EDIT
```

```suggestion
    Telecrystal: 14 # WD EDIT
```
Раз уж ему дали полную защиту от огня (Хотя зачем? У нас нету огнемета)